dev/NavigationView/NavigationView.xaml

```diff
@@ -18,36 +18,9 @@
             <Setter.Value>
                 <ControlTemplate TargetType="local:NavigationView">
                     <Grid x:Name="RootGrid">
-                        <Grid.Resources>
-                            <Storyboard x:Name="ShadowCasterEaseInStoryboard">
```

Comment:
@ranjeshj Is this safe to delete? technically it is a named resource..

Reply:
@MikeHillberg FYI

---
In reply to: [605142979](https://github.com/microsoft/microsoft-ui-xaml/pull/4659#discussion_r605142979) [](ancestors = 605142979)

Reply:
That was the storyboard I added for the NavigationView Shadow PR, I don't know if that was checked in for an actual release or preview though. However, I moved it into `PaneOverlayGroup`'s VisualTransition bit and kept the name just in case. Should that be removed?

Reply:
ah, okay I think that's okay then